content/usage/advanced/index.md

```diff
@@ -205,6 +222,8 @@ Submit feedback about BlueOS via:
 - [Issues on the GitHub repository](https://github.com/bluerobotics/BlueOS/issues)
    - allows easily tracking changes, 
 and notification when complete/fixed
+- Report form submissions
+   - Can be submitted at any time, and get sent to the developers when the vehicle next connects to the internet
```

Comment:
surface computer is connected to the internet

Reply:
Ahh, makes sense, thanks :-)